liveblog.php

```diff
@@ -1275,6 +1609,19 @@ private static function is_wp_too_old() {
 		}
 		return version_compare( $wp_version, self::min_wp_version, '<' );
 	}
+
+	/**
+	 * Returns refresh interval after filters have been run
+	 *
+	 * @return int
+	 */
+	public static function get_refresh_interval() {
```

Comment:
Nice!

Reply:
@philipjohn does this require any action?